tanjun/abc.py

```diff
@@ -362,6 +423,36 @@ async def edit_initial_response(
             typing.Union[hikari.SnowflakeishSequence[hikari.PartialRole], bool]
         ] = hikari.UNDEFINED,
     ) -> hikari.Message:
+        """Edit the initial response for this context.
+
+        Parameters
+        ----------
+        content : hikari.UndefinedOr[typing.Any]
+            The content to edit the response with.
+
+        Other Parameters
+        ----------------
+        attachment : hikari.UndefinedOr[hikari.Resourceish]
+            A singular attachment to edit the response with.
+        attachments : hikari.UndefinedOr[collections.Sequence[hikari.Resourceish]]
+            A sequence of attachments to edit the response with.
+        embed : hikari.UndefinedOr[hikari.Embed]
+            An embed to replace the response with.
+        embeds : hikari.UndefinedOr[collections.Sequence[hikari.Embed]]
+            A sequence of embeds to replace the response with.
+        replace_attachments : bool
+            Whether to replace the attachments of the response or not. Default to `False`.
+        mentions_everyone : hikari.UndefinedOr[bool]
+            If set to `True`. The response will parse the @everyone/@here mentions. Default is `hikari.UNDEFINED`
+        user_mentions : hikari.UndefinedOr[typing.Union[hikari.SnowflakeishSequence[hikari.PartialUser], bool]]
+            If set to `True`, all user mentions will be detected.
+            If set to `False`, all user mentions will be ignored.
```

Comment:
These segments seem to be missing a bit about specifying specific users/roles to mention